src/EditorFeatures/Core/Implementation/Peek/PeekableItemSource.cs

```diff
@@ -67,6 +67,11 @@ public void AugmentPeekSession(IPeekSession session, IList<IPeekableItem> peekab
                     // For documents without semantic models, just try to use the goto-def service
                     // as a reasonable place to peek at.
                     var goToDefinitionService = document.GetLanguageService<IGoToDefinitionService>();
+                    if (goToDefinitionService == null)
```

Comment:
If there's not too many other warnings, could you also `nullable enable` this file?

Reply:
Wasn't too bad - enabled and fixed it up.